docs/understanding-airbyte/glossary.md

```diff
@@ -0,0 +1,54 @@
+# Glossary of Terms
+
+### ETL/ELT
+Stands for **E**xtract, **T**ransform, and **L**oad and **E**xtract, **L**oad, and **T**ransform, respectively.
+
+**Extract**: Retrieve data from a [source](../integrations/sources), which can be an application, database, anything really.
+
+**Transform**: Clean up the data. This is referred to as [normalization](./basic-normalization.md) in Airbyte and involves [deduplication](./connections/incremental-deduped-history.md), changing data types, formats, and more.
+
+**Load**: Move data to your [destination](../integrations/destinations).
+
+### Airbyte CDK
+
+The Airbyte CDK (Connector Development Kit) allows you to create connectors for Sources or Destinations. If your source or destination doesn't exist, you can use the CDK to make the building process a lot easier. It generates all the tests and files you need and all you need to do is write the connector-specific code for your source or destination. We created one in Python which you can check out [here](../connector-development/cdk-python) and the Faros AI team created a Javascript/Typescript one that you can check out [here](../connector-development/cdk-faros-js).
+
+### Full Refresh Sync
+
+A **Full Refresh Sync** will attempt to retrieve all data from the destination every time a sync is run. Then there are two choices, **Overwrite** and **Append**. **Overwrite** deletes the data in the destination before running the sync and **Append** doesn't.
+
+### Incremental Sync
+
+An **Incremental Sync** will only retrieve new data everytime the a sync occurs. The first sync will always attempt to retrieve all the data. If the [destination supports it](https://discuss.airbyte.io/t/what-destinations-support-the-incremental-deduped-sync-mode/89), you can have your data deduplicated. Simply put, this just means that if you sync an updated version of a record you've already synced, it will remove the old record.
```

Comment:
```suggestion
An **Incremental Sync** will only retrieve new data from the source every time the a sync occurs. The first sync will always attempt to retrieve all the data. If the [destination supports it](https://discuss.airbyte.io/t/what-destinations-support-the-incremental-deduped-sync-mode/89), you can have your data deduplicated. Simply put, this just means that if you sync an updated version of a record you've already synced, it will remove the old record.
```